### ISSUE TITLE
Bezier curves can go through obstacles

### ISSUE THREAD

**Issue by joshhting** [RoboJackets/rrt#18]

**Comment by justbuchanan** [comment]
Yeah this is a known issue.  It's because the bezier path is applied on top of the line segment path generated by the RRT.  The segmented path is obstacle-free, but the bezier is often not.  I'm hoping to resolve this by limiting the angle of each new path added to the RRT so the path generated from that will be closer to the bezier put on top of it.

**Comment by joshhting** [comment]
In order to help compensate for that, could we have the rrt act as if the obstacles are closer than they appear? This would prevent the rrt from placing nodes too close to the obstacles that could risk the bezier curve touching them.
